.circleci/invfile.lua

```diff
@@ -0,0 +1,146 @@
+-- http://stackoverflow.com/questions/19262761/lua-need-to-split-at-comma/19263313#19263313
```

Comment:
This file is a copy from the one coming with `galaxy-util-tool` but uses `-lc` instead of `-c` as options for all `/bin/sh` calls. The `-l` option tells the shell to use a login session and this leads to the loading of `/etc/profile` and `pam_umask.so` that sets the `umask` to `0022`.